templates/html-end.php

```diff
@@ -19,7 +19,17 @@
  */
 ?>
 
-<?php do_action( 'amp_post_template_footer', $this ); ?>
+<?php
+/**
+ * Fires when just before after printing the </body>.
```

Comment:
Huh? 🤔

Reply:
Extraneous “after“.

Reply:
More than that... probably extraneous "when" as well.